src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -135,15 +135,15 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
     /**
      * {@inheritdoc}
      */
-    public function supportsNormalization(mixed $data, string $format = null): bool
+    public function supportsNormalization(mixed $data, string $format = null)
```

Comment:
The return type has to be re-added to the PHPDoc block, here and for all other methods changed in this file.

Reply:
we don't have this one on 5.4 so it's not needed

Reply:
All right.